acceptance/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:14.04
 MAINTAINER James C. Scott III <jcscott.iii@gmail.com>
 
+RUN apt-get -qq update
```

Comment:
@jcscottiii this fixed the build problems I was having in the main convo, but would like your feedback on it.